src/test/java/io/debezium/connector/vitess/VitessConnectorIT.java

```diff
@@ -820,15 +855,24 @@ private void startConnector(boolean hasMultipleShards) throws InterruptedExcepti
     private void startConnector(Function<Configuration.Builder, Configuration.Builder> customConfig,
                                 boolean hasMultipleShards)
             throws InterruptedException {
-        startConnector(customConfig, hasMultipleShards, false, 1, -1, -1, null);
+        startConnector(customConfig, hasMultipleShards, false, 1, -1, -1, null, VitessConnectorConfig.SnapshotMode.NEVER);
     }
 
     private void startConnector(Function<Configuration.Builder, Configuration.Builder> customConfig,
                                 boolean hasMultipleShards, boolean offsetStoragePerTask,
                                 int numTasks, int gen, int prevNumTasks, String tableInclude)
             throws InterruptedException {
+        startConnector(customConfig, hasMultipleShards, offsetStoragePerTask, numTasks, gen, prevNumTasks,
+                tableInclude, VitessConnectorConfig.SnapshotMode.NEVER);
+    }
+
+    private void startConnector(Function<Configuration.Builder, Configuration.Builder> customConfig,
+                                boolean hasMultipleShards, boolean offsetStoragePerTask,
+                                int numTasks, int gen, int prevNumTasks, String tableInclude,
+                                VitessConnectorConfig.SnapshotMode snapshotMode)
```

Comment:
[memo] When you give `null` as snapshotMode here, the connector uses the default snapshotMode (INITIAL).